packages/@lightningjs/ui-components/src/components/Tile/Tile.d.ts

```diff
@@ -18,10 +18,20 @@
 
 import lng from '@lightningjs/core';
 import { MetadataBaseStyle } from '../MetadataBase';
-import type { StylePartial } from '../../types/lui';
-import Surface, { SurfaceStyle } from '../Surface';
+import { StylePartial } from '../../types/lui';
+import { SurfaceStyle } from '../Surface';
+import {
+  Artwork,
+  TextBox,
+  ProgressBar,
+  MetadataBase,
+  Badge,
+  Checkbox,
+  Label,
+  Surface
+} from '..';
```

Comment:
as a rule we're not using intermediate imports in LUI, these should each be imported separately 😬 sorry